client/campaign_message_configuration/README.md

```diff
@@ -16,51 +16,26 @@ Linux or Windows
 King Phisher Minimum Required Version = 1.10.0 
 ```
 
-### Installing 
+### Use King Phisher Plugin Manager to install and enable plugin
 
 - [ ] Download plugin 
-- [ ] Move plugin to: <code>/opt/king-phisher-plugins/client/</code>
-- [ ] Configure plugin<code>(refer to Deployment & Configuration below)</code> 
+- [ ] Move plugin to: `/opt/king-phisher-plugins/client/`

```

Comment:
if manually installing the plugin in needs to be placed into `~/.config/king-phisher/plugins`

Reply:
change to
`[ ] Use King Phisher Clients plugin manager to install and enable`